metadata.json

```diff
@@ -7,6 +7,35 @@
   "source": "git://github.com/Ericsson/puppet-module-cron.git",
   "project_page": "https://github.com/Ericsson/puppet-module-cron",
   "issues_url": "https://github.com/Ericsson/puppet-module-cron/issues",
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "6"
```

Comment:
Sharp eyes @ghoneycutt :)

Looking at the history I can see that Ubuntu 12 was meant. In one commit [1] $osfamily got fixed to contain Debian (which is right), too bad this commit also changed the comments from Ubuntu to Debian.

@NissesSenap: this should read like this instead

```
    {
      "operatingsystem": "Ubuntu",
      "operatingsystemrelease": [
        "12.04"
      ]
    }
```

[1] https://github.com/Ericsson/puppet-module-cron/commit/403bbd2958081f8005f1f81511192f0e887a762d